fpdc/test_settings.py

```diff
@@ -0,0 +1,3 @@
+from .settings import *
```

Comment:
Would it make more sense to convert `fpdc.settings` into a module, and define `test` as a submodule? (Similar to how [`happiness-packets`](https://pagure.io/fedora-commops/fedora-happiness-packets/blob/master/f/happinesspackets/settings) does it.)

Reply:
I don't have any preferences, is there any benefit of having a settings module ? I guess on the long term it is easier to organize.

Reply:
I think mainly it reduces file clutter. All settings end up under `appname/settings/` and can include the contents of `appname/settings/__init__.py`.

Reply:
ok I ll update that :dancer: